nemo/deploy/nlp/megatronllm_deployable.py

```diff
@@ -142,6 +157,11 @@ def _load_from_nemo_checkpoint(self, nemo_checkpoint_filepath: str, num_devices:
             # had to override these to make Nemotron3-22B work, see sample_sequence_batch() in text_generation_utils.py
             custom_config.activations_checkpoint_granularity = None
             custom_config.activations_checkpoint_method = None
+            custom_config.dist_ckpt_load_strictness = StrictHandling.LOG_ALL.value
```

Comment:
@thomasdhc @mikolajblaz @dimapihtar do you have any idea why the 1st error mentioned in MR description is visible in **r2.0.0** branch but on the other hand **main** looks good with the same repro?

Reply:
Discussed with @mikolajblaz offline, this is likely due to different TE versions used in two different containers tested

Reply:
correct. If you  don't want to import MCore you can set a string `'log_all'` here

Reply:
I think it's fine to import it as it's required here anyway. This is also more transparent to me besides on what's going on.